### PR TITLE
feat(ams-ui): added table for showing crop information

### PR DESCRIPTION
**Pull request by aashnaPrakash**
- added table to show crop info 
- added mock data for crop growth protocol (for testing purposes)
- fixed css issues related to column
- added validation to avoid calling aPI when user does not select a locality while fetching fields

### CODE REVIEW

File: src/Pages/CropInfoPage.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+ * Component to show Crop Information
+ *
+ * @returns Table to show crop info
+ */
+function CropInfoPage() {
+  const classes = useStyles();
+  const [alertMessage, setAlertMessage] = useState('');
+  const [alertStatus, setAlertStatus] = useState(false);
+  const [alertSeverity, setAlertSeverity] = useState('');
+  const [cropsData, setCrops] = useState([]);
```
Comment (gautamguptabasant) [inline src/Pages/CropInfoPage.js]: cropsData -> crops

File: src/Pages/FieldInfoPage.js [hunk 2]
```diff
@@ -222,6 +229,9 @@ function FieldInfoPage() {
             );
           })}
         </Select>
+        {showLocalityError ? (
+          <FormHelperText>Select a locality</FormHelperText>
+        ) : null}
```
Comment (gautamguptabasant) [inline src/Pages/FieldInfoPage.js]: Can do `showLocalityError && (<FormHelperText>Select a locality</FormHelperText>)` for brevity?

**Review by gautamguptabasant — COMMENTED** [review]
Looks good, minor comments.

File: src/Pages/CropInfoPage.js (hunk 1 above)
Comment (aashnaPrakash) [reply src/Pages/CropInfoPage.js]: done

File: src/Pages/FieldInfoPage.js (hunk 2 above)
Comment (aashnaPrakash) [reply src/Pages/FieldInfoPage.js]: done

**Review by gautamguptabasant — APPROVED** [review]

File: src/Pages/FieldInfoPage.js [hunk 3]
```diff
@@ -154,17 +156,22 @@ function FieldInfoPage() {
...
    * shows alert in case call fails
    */
   const fetchFieldsForLocation = () => {
-    getFieldsByLocation(locality)
-      .then(setFields)
-      .catch((e) => {
-        console.log(`Fetching fields for ${locality} failed`, e);
-        showAlert(`Fetching fields for ${locality} failed`, 'error');
-      });
+    if (Number.isInteger(locality) {
```
Comment (mahimavuppuluri) [inline src/Pages/FieldInfoPage.js]: missing a bracket here

**Pull request merged**